web3/engine_api_types.nim

```diff
@@ -47,11 +47,11 @@ type
 
 const
   # https://github.com/ethereum/execution-apis/blob/v1.0.0-beta.1/src/engine/specification.md#errors
-  engineApiParseError* = - 32700
-  engineApiInvalidRequest* = -32600
-  engineApiMethodNotFound* = -32601
-  engineApiInvalidParams* = -32602
-  engineApiInternalError* = -32603
-  engineApiServerError* = -32000
-  engineApiUnknownPayload* = -38001
-  engineApiInvalidPayloadAttributes* = -38002
+  engineApiParseError*                = -32700
```

Comment:
generally, these kind of alignments are messy to maintain and lead to large unnecessary git diffs

Reply:
Yep, completely agree. Actually I noticed that we don't use `nimpretty` to auto-format the code?

Reply:
`nimpretty` often makes code more ugly and sometimes breaks (it's too buggy to use in production code) - the only automatic formatting we apply is to remove trailing whitespace.